app/src/main/java/org/mozilla/focus/locale/LocaleManager.java

```diff
@@ -283,6 +283,16 @@ public Locale getCurrentLocale(@NonNull Context context) {
         return currentLocale = Locales.parseLocaleCode(current);
     }
 
+    public boolean currentLanguageIsEnglish(@NonNull Context context) {
+        Locale current = getCurrentLocale(context);
+        // If locale hasn't been updated (i.e., 'current' is null), use system default
+        if (current == null) {
+            current = context.getResources().getConfiguration().locale;
```

Comment:
I'm not sure why this file has so many ways to get locales, but this seems reasonable to get the system locale (from looking at other methods here).

Reply:
There's a pretty crazy comment on line 89 above a line using this, apparently other code in this class mutates some other ways of getting the locale.